recipes/LibriSpeech/ASR/seq2seq/hparams/train_BPE_1000.yaml

```diff
@@ -44,7 +44,7 @@ test_csv:
 # Training parameters
 number_of_epochs: 15
 number_of_ctc_epochs: 5
-batch_size: 8
+batch_size: 4
```

Comment:
Where the heck do these changes come from. Changing this will make the corresponding drive model change. Someone tested those params?